src/pb_env.cc

```diff
@@ -253,6 +253,18 @@ EnvironmentManager::ExtractIfNotExtracted(std::string env_path)
 
   bool env_extracted = false;
   bool re_extraction = false;
+
+  // If the path is not a conda-packed file, then bypass the extraction process
+  std::string subPath = env_path.substr(env_path.size() - 6);
+  if (subPath != "tar.gz") {
+    LOG_MESSAGE(
+        TRITONSERVER_LOG_VERBOSE,
```

Comment:
This might be useful to have at the info-level logging level.

Reply:
I'm keeping this inline with the other messages. See line 287